src/ruby/README.md

```diff
@@ -50,7 +50,19 @@ $ # from this directory
 $ bundle install  # creates the ruby bundle, including building the grpc extension
 $ rake  # runs the unit tests, see rake -T for other options
 ```
-
+- If you are having trouble running the unit tests, try the following
+  1. If you see an error message similar to the one below
+     ```sh
+     Failure/Error: fail 'CONFIG env variable unexpectedly unset' unless ENV['CONFIG']
+     ```
+     Try the following:
+     ```sh
+     export CONFIG=opt # set CONFIG environment variable
```

Comment:
CONFIG could be set to opt (the default build config), but it could also be set to "dbg", depending on whether on what mode the `grpc_ruby_plugins` make target was built under ("dbg" would be the case if previously ran `CONFIG=dbg make gprc_ruby_plugins)`. I might add that a pre-req here is that `make grpc_ruby_plugins needs to have been ran beforehand, which `rake` doesn't know about (I'm not sure if there could actually be a good way to have this target get built by the Rakefile as a part of running tests) - ultimately, this project's "canonical test runner" is `tools/run_tests/run_tests.py -l ruby`, rather than "rake", which is a gotcha.